tests/acceptance/features/webUISharingInternalGroups/shareWithGroups.feature

```diff
@@ -287,13 +287,14 @@ Feature: Sharing files and folders with internal groups
     Then user "User Two" should be listed as "Editor" via "simple-folder" in the collaborators list on the webUI
     And group "grp1" should be listed as "Editor" via "sub-folder" in the collaborators list on the webUI
 
-  @skip @yetToImplement @issue-2898
+  @issue-2898
```

Comment:
does that issue still apply to the test?

Reply:
I wanted to keep it for easy reference and being able to run all tests matching this issue/work

let me know if this is not ok